File: internal/env/env.go
package env

import (
	"errors"
	"fmt"
	"os"
)

func GetString(key string) (string, error) {
	val, ok := os.LookupEnv(key)
	if !ok {
		fmt.Printf("Could not find key: %s\n", key)
		return "", errors.New("Could not find key: " + key)
	}
	return val, nil
}

File: app/env.py
"""Port of internal/env/env.go.

Note: just like the Go version, main.py does NOT use this helper - it reads
os.environ directly with its own error handling. Kept as-is (unused) for
fidelity to the original.
"""

import os


def get_string(key: str) -> str:
    val = os.environ.get(key)
    if val is None:
        print(f"Could not find key: {key}")
        raise KeyError(f"Could not find key: {key}")
    return val
